source/global.cc

```diff
@@ -102,4 +102,30 @@ namespace aspect
 
   // Output parallel statistics [bool]
   const bool output_parallel_statistics = false;
+
+
+  namespace citation
+  {
+    std::vector<std::string> citations;
+
+    std::string get_url ()
+    {
+      std::string args = "";
+      for (unsigned int i=0; i<citations.size(); ++i)
+        {
+          args += citations[i]+"=1";
```

Comment:
I accidentally read this line as having *two* `operator+=` uses. Maybe add spaces around the plus operator.